### ISSUE TITLE
Logitech M535 support

### ISSUE THREAD

**Issue by GianMantuan** [libratbag/libratbag#656]
Hello, i'm trying to get this device to work in my system and I managed to get the pID = 046d / vID = b014.
I created a .device file under the /usr/local/share/libratbag/devices 

[Device]
Name=Logitech M535
DeviceMatch=usb:046d:b014
Driver=hidpp20

Since this device is bluetooth, i tried with the bus flag as bluetooth but didn't work, and changing to usb also didn't work.
Is there a WIP for this device or how can I help to get this to work

**Comment by Hjdskes** [comment]
Did you restart `ratbagd` after adding the new .device file?

**Comment by GianMantuan** [comment]
> Did you restart `ratbagd` after adding the new .device file?

Yes, the Piper cannot find any devices, and running `ratbagctl` returns "No devices available"

**Comment by GabMus** [comment]
Also interesting in getting libratbag support for this mouse. It's not essential but it'd be nice to be able to remap some buttons.
